create-hpc-cluster-linux-cn/azuredeploy.json

```diff
@@ -200,99 +150,132 @@
         "publisher": "OpenLogic",
         "offer": "CentOS",
-        "sku": "7.2"
+        "sku": "7.2",
+        "version": "latest"
+      },
+      "CentOS_7.3": {
+        "publisher": "OpenLogic",
+        "offer": "CentOS",
+        "sku": "7.3",
+        "version": "latest"
       },
       "CentOS_6.5_HPC": {
         "publisher": "OpenLogic",
         "offer": "CentOS-HPC",
-        "sku": "6.5"
+        "sku": "6.5",
+        "version": "latest"
+      },
+      "CentOS_6.8_HPC": {
+        "publisher": "OpenLogic",
+        "offer": "CentOS-HPC",
+        "sku": "6.8",
+        "version": "latest"
       },
       "CentOS_7.1_HPC": {
         "publisher": "OpenLogic",
         "offer": "CentOS-HPC",
-        "sku": "7.1"
+        "sku": "7.1",
+        "version": "latest"
+      },
+      "CentOS_7.3_HPC": {
+        "publisher": "OpenLogic",
+        "offer": "CentOS-HPC",
+        "sku": "7.3",
+        "version": "latest"
       },
       "RHEL_6.7": {
         "publisher": "RedHat",
         "offer": "RHEL",
-        "sku": "6.7"
+        "sku": "6.7",
+        "version": "latest"
       },
       "RHEL_6.8": {
         "publisher": "RedHat",
         "offer": "RHEL",
-        "sku": "6.8"
+        "sku": "6.8",
+        "version": "latest"
       },
       "RHEL_7.2": {
         "publisher": "RedHat",
         "offer": "RHEL",
-        "sku": "7.2"
+        "sku": "7.2",
+        "version": "latest"
+      },
+      "RHEL_7.3": {
+        "publisher": "RedHat",
+        "offer": "RHEL",
+        "sku": "7.3",
+        "version": "latest"
       },
-      "SLES_12": {
+      "SLES_12_SP2": {
         "publisher": "SUSE",
         "offer": "SLES",
-        "sku": "12"
+        "sku": "12-SP2",
+        "version": "latest"
       },
-      "SLES_12_Premium": {
+      "SLES_12_SP2_Premium": {
         "publisher": "SUSE",
         "offer": "SLES-Priority",
-        "sku": "12"
-      },
-      "SLES_12_HPC": {
-        "publisher": "SUSE",
-        "offer": "SLES-HPC",
-        "sku": "12"
-      },
-      "SLES_12_HPC_Premium": {
-        "publisher": "SUSE",
-        "offer": "SLES-HPC-Priority",
-        "sku": "12"
+        "sku": "12-SP2",
+        "version": "latest"
       },
       "Ubuntu_14.04": {
         "publisher": "Canonical",
         "offer": "UbuntuServer",
-        "sku": "14.04.3-LTS"
+        "sku": "14.04.3-LTS",
+        "version": "latest"
       },
       "Ubuntu_16.04": {
         "publisher": "Canonical",
         "offer": "UbuntuServer",
-        "sku": "16.04.0-LTS"
+        "sku": "16.04.0-LTS",
+        "version": "latest"
       }
     },
     "currentComputeNodeImage": "[variables('computeNodeImages')[parameters('computeNodeImage')]]",
-    "artifactsBaseUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/create-hpc-cluster"
+    "artifactsBaseUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/create-hpc-cluster/shared-resources"
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc) for how to handle artifacts